### PR TITLE
feat(UIForm): pass event to onChange/onTrigger callbacks

### PR DESCRIPTION
**Pull request by jsomsanith**
**What is the problem this PR is trying to solve?**
The onChange and onTrigger callbacks has business arguments, but :
* event is missing
* it waits for 3/4 arguments, in a particular order, which is a hell to remember
* onTrigger and onChange has different payloads

**What is the chosen solution to this problem?**
* Align the callbacks arguments : callback(event, payload)
* Align (as possible) the payloads

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/forms/src/UIForm/UIForm.component.js [hunk 1]
```diff
@@ -43,19 +43,19 @@ export default class UIForm extends React.Component {
 	 * @param schema The field schema
 	 * @param value The new value
 	 */
-	onChange(event, schema, value) {
+	onChange(event, { schema, value }) {
```
Comment (acateland) [inline packages/forms/src/UIForm/UIForm.component.js]: can you please update jsdoc

File: packages/forms/src/UIForm/UIForm.component.js [hunk 2]
```diff
@@ -43,19 +43,19 @@ export default class UIForm extends React.Component {
...
 			customValidation,
 		} = this.props;
 		const error = validateValue(schema, value, properties, customValidation);
-		onChange(formName, schema, value, error);
+		onChange(event, { formName, schema, value, error });
 
 		const { triggers } = schema;
 		if (triggers && triggers.includes(TRIGGER_AFTER)) {
-			this.onTrigger(event, TRIGGER_AFTER, schema, value, properties);
+			this.onTrigger(event, { type: TRIGGER_AFTER, schema, value, properties });
```
Comment (acateland) [inline packages/forms/src/UIForm/UIForm.component.js]: why not giving formName ? Why parameters are so différents between onChange and onTrigger callback

Comment (jsomsanith) [inline packages/forms/src/UIForm/UIForm.component.js]: Aligned payloads

File: packages/forms/src/UIForm/UIForm.component.js (hunk 1 above)
Comment (jsomsanith) [inline packages/forms/src/UIForm/UIForm.component.js]: done

**Pull request merged**